examples/Ballot.bl

```diff
@@ -0,0 +1,89 @@
+version 1.1
+
+contract Ballot {
+
+    struct Voter {
+        int weight
+        bool voted
+        address delegate
+        int vote
+    }
+
+    struct Proposal {
+        String name
+        int voteCount
+    }
+
+    address chairperson
+
+    Map<address, Voter> voters
+
+    Proposal[] proposals
+
+    constructor(String[] proposalNames) {
+        chairperson = msg.sender
+        voters[chairperson].weight = 1
+        proposals = new Proposal[proposalNames.length]
```

Comment:
do we need a "new" keyword and do we support dynamic length?

Reply:
`new` keyword is required to differentiate between index access and array creation in Parser.
We don't support dynamic length. Is `slice/list` required?